setup-system.sh

```diff
@@ -1,5 +1,10 @@
 set -e
 
+# do an update to start things off
+function apt_update() {
+    sudo apt-get update
```

Comment:
That's unnecessarily passive aggressive when you're proposing merges into my project. How about something like "You also have an apt-get on line  17, just to let you know"?